flavors/integration_test/flavor_base/build_steps.py

```diff
@@ -20,6 +20,18 @@ def get_path_in_flavor(self):
         return "flavor_base"
 
 
+class AnalyzeFlavorTestBuildRun(DockerFlavorAnalyzeImageTask):
+
+    def get_build_step(self) -> str:
+        return "flavor_test_build_run"
+
+    def requires_tasks(self):
+        return {"base_test_build_run": AnalyzeBaseTestBuildRun}
+
+    def get_path_in_flavor(self):
+        return "flavor_base"
```

Comment:
Needed, otherwise exaslct build runs in the following error:
```
...
  File "/home/runner/.cache/pypoetry/virtualenvs/exasol-script-languages-container-ci-setup-Esv2GY0r-py3.10/lib/python3.10/site-packages/exasol_integration_test_docker_environment/lib/docker/images/create/docker_build_base.py", line 67, in create_analyze_tasks
    raise Exception(
Exception: Unknown goal(s) set(), following goals are avaialable {'security_scan', 'base_test_build_run', 'release'}
```

Reply:
Oh, that sounds like a bug

Reply:
I think, we shouldn't fail when a default goal is not available.

Reply:
This is the comparison that causes it https://github.com/exasol/integration-test-docker-environment/blob/ee3bc87d74d4b06800fb5e9fd5f9e8a59181d07b/exasol_integration_test_docker_environment/lib/docker/images/create/docker_build_base.py#L62